src/Views/DataView.php

```diff
@@ -89,6 +92,20 @@ public function getItems(Searchable $searchable, Filterable $filterable, Sortabl
         return $query->paginate($this->paginate);
     }
 
+    public function applyDefaultFilters()
+    {
+        if(empty($this->filters))
+        {
+            foreach($this->filters() as $filter)
```

Comment:
What if we have multiple filters with a default value? Will the last one be assigned?